dynawo/sources/API/TL/TLEventImpl.h

```diff
@@ -20,6 +20,7 @@
 #ifndef API_TL_TLEVENTIMPL_H_
 #define API_TL_TLEVENTIMPL_H_
 
+#include <typeinfo>
```

Comment:
I don't see any use of typeid that would require this include, except if there is any other method or operator requiring it that I haven't found

Reply:
you are right, I removed it.